### PR TITLE
Mat 54 newsletter mutation

### PR DESCRIPTION
**Pull request by haylau**
Create tests passed

### CODE REVIEW

**Comment by haylau** [comment]
+ table "NewsletterEmailModel" deployed

**Comment by haylau** [comment]
image

**Comment by haylau** [comment]
+ includes changes to a typo made (no one caught :despair:) to postmodel mutations

**Review by mohazahid — COMMENTED** [review]
LGTM

**Review by mohazahid — DISMISSED** [review]
LGTM 2

File: amplify/backend/api/Unhinged/schema.graphql [hunk 1]
```diff
@@ -1,3 +1,8 @@
+type NewsletterEmailModel @model @auth(rules: [{allow: public}]) {
```
Comment (akumar282) [inline amplify/backend/api/Unhinged/schema.graphql:1]: Permissions may need to be changed on this model

File: src/backend/types.ts [hunk 2]
```diff
@@ -19,14 +22,27 @@ export type DeleteUserModelPayload = {
...
+  input: CreateNewsletterEmailModelInput
+}
+
+export type UpdateewsletterEmailModelPayload = {
+  input: UpdateNewsletterEmailModelInput
+}
+
+export type DeleteewsletterEmailModelPayload = { 
+  input: DeleteNewsletterEmailModelInput
+}
```
Comment (akumar282) [inline src/backend/types.ts:47]: I think you meant UpdateNews, and DeleteNews

File: test/backend/mutations/newsletterEmailMutations.test.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+describe('Mutation Tests', () => {
+
+  test('Create NewsletterEmail', async () => {
+    const result = await createNewsletterEmail(testload)
+    const flatResult = JSON.stringify(result)
+    expect(flatResult).toContain(`unhinged@matchbox.com`)
+  })
+
+})
```
Comment (akumar282) [inline test/backend/mutations/newsletterEmailMutations.test.ts:22]: I'm now thinking that we should put all API call tests in one place and only sort by what type of call it is. Consolidate all queries and mutations. But that might be out of scope for this PR

**Review by akumar282 — CHANGES_REQUESTED** [review]

**Review by akumar282 — DISMISSED** [review]

**Review by mohazahid — APPROVED** [review]
LGTM

**Review by akumar282 — APPROVED** [review]
TODO: Make ticket for code cleanup

**Comment by haylau** [comment]
> TODO: Make ticket for code cleanup

*test code cleanup

**Pull request merged**